### PR TITLE
Document trustedProxy variable in config.json

### PR DESCRIPTION
**Pull request by keedya**
depends on: https://github.com/RackHD/on-http/pull/400#issuecomment-237597769

@zyoung51 @heckj @RackHD/corecommitters

### CODE REVIEW

File: docs/rackhd/configuration.rst [hunk 1]
```diff
@@ -166,6 +167,8 @@ The following table describes the configuration parameters in config.json:
       - Address for RackHD WebSocket Service to bind to (defaults to '0.0.0.0').
     * - wssBindPort
       - Listening port for RackHD WebSocket Service (defaults to 9100).
+    * - trustedProxy
```
Comment (heckj) [inline docs/rackhd/configuration.rst]: dumb question - what's this default to? (i.e. what value is used if it's not explicitly in the monorail.json configuration file)

Comment (keedya) [inline docs/rackhd/configuration.rst]: defaults to False

File: docs/rackhd/configuration.rst [hunk 2]
```diff
@@ -166,6 +167,8 @@ The following table describes the configuration parameters in config.json:
...
+      - Enable trust proxy in express. Populate req.ip with most left in XforwardFor list
```
Comment (zyoung51) [inline docs/rackhd/configuration.rst]: Worth adding a link to ExpressJS documentation here? https://expressjs.com/en/guide/behind-proxies.html

**Comment by zyoung51** [comment]
+1

**Comment by benbp** [comment]
:

**Pull request merged**